content/departments/product-engineering/engineering/enablement/dev-experience/processes.md

```diff
@@ -35,18 +35,31 @@ We collect meeting notes and planning artifacts in a shared [Google Drive folder
 
 If you have a question that relates to our [areas of ownership](../../process/engineering_ownership.md), the #dev-experience channel is the best way to reach us. Tag the `@dev-experience-support` handle if something is urgent (think of this as pulling the fire alarm) and requires our immediate attention. Our on-call teammate will jump in.
 
-### Raising an issue
+#### Raising an issue
 
-If you think Dev Experience is the right team to address a bug or other request, please create an issue. The issue should include:
+If you think Dev Experience is the right team to address a bug or other request, please [create an issue](https://github.com/sourcegraph/sourcegraph/issues). The issue should include:
 
 - A short description of the ask
 - A more detailed explanation of the background, context, and problem that needs solving
 - Any guidance related to the impact of this issue
 - Any extra information that will help us solve or prioritize the issue
 - The label `team/devx`
 
+### Providing support
+
+This section documents tips for teammates on support, which is handled via our [support rotation](#support-rotation).
+General tips:
+
+1. We are not responsible for every issue in CI - see [build pipeline support](#build-pipeline-support) and [CI support responsibilities](#ci-support-responsibilities).
+2. `git rebase` is very easy for teammates unfamiliar with `git` to do incorrect - if a branch update is required, err towards recommending a [`git merge`](https://git-scm.com/docs/git-merge) (e.g. `git pull origin main`).
```

Comment:
```suggestion
2. `git rebase` is very easy for teammates unfamiliar with `git` to do incorrect - if a branch update is required, err toward recommending a [`git merge`](https://git-scm.com/docs/git-merge) (e.g. `git pull origin main`).
```

Not sure about the grammar here, but just in case ⬆️

Reply:
There's an S! https://www.macmillandictionary.com/dictionary/british/err-towards-in-favour-of-something#:~:text=DEFINITIONS1,outsider%20for%20the%20top%20job.